samples/host_verify/README.md

```diff
@@ -0,0 +1,69 @@
+
+# The host-side enclave verification sample
+
+- Demonstrates the attestation of a remote enclave
+- Requires only the `openenclave/host_verify.h` header, as well as some C standard header files
+- Also runs in simulation mode and when both BUILD_ENCLAVES and HAS_QUOTE_PROVIDER flags are turned off
```

Comment:
This is misleading since host verification is targeted for non  SGX machines.

Reply:
Ah, good point. Just removed this line as well as the simulation part in CMakeLists.txt.